internal/blockchain/fabric/fabric_provider.go

```diff
@@ -250,18 +295,46 @@ func (p *FabricProvider) extractChaincode() error {
 	return nil
 }
 
-func (p *FabricProvider) installChaincode() error {
+func (p *FabricProvider) installChaincode(packageFilename string) error {
 	p.Log.Info("installing chaincode")
 	stackDir := path.Join(constants.StacksDir, p.Stack.Name)
 	volumeName := fmt.Sprintf("%s_firefly_fabric", p.Stack.Name)
-	return docker.RunDockerCommand(stackDir, p.Verbose, p.Verbose, "run", "--platform", getDockerPlatform(), "--rm", fmt.Sprintf("--network=%s_default", p.Stack.Name), "-e", "CORE_PEER_ADDRESS=fabric_peer:7051", "-e", "CORE_PEER_TLS_ENABLED=true", "-e", "CORE_PEER_TLS_ROOTCERT_FILE=/etc/firefly/organizations/peerOrganizations/org1.example.com/peers/fabric_peer.org1.example.com/tls/ca.crt", "-e", "CORE_PEER_LOCALMSPID=Org1MSP", "-e", "CORE_PEER_MSPCONFIGPATH=/etc/firefly/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp", "-v", fmt.Sprintf("%s:/contracts", path.Join(stackDir, "contracts")), "-v", fmt.Sprintf("%s:/etc/firefly", volumeName), "hyperledger/fabric-tools:2.3", "peer", "lifecycle", "chaincode", "install", "/contracts/firefly_fabric.tar.gz")
+	return docker.RunDockerCommand(stackDir, p.Verbose, p.Verbose,
+		"run",
+		"--platform", getDockerPlatform(),
+		"--rm",
+		fmt.Sprintf("--network=%s_default", p.Stack.Name),
+		"-e", "CORE_PEER_ADDRESS=fabric_peer:7051",
+		"-e", "CORE_PEER_TLS_ENABLED=true",
+		"-e", "CORE_PEER_TLS_ROOTCERT_FILE=/etc/firefly/organizations/peerOrganizations/org1.example.com/peers/fabric_peer.org1.example.com/tls/ca.crt",
+		"-e", "CORE_PEER_LOCALMSPID=Org1MSP",
+		"-e", "CORE_PEER_MSPCONFIGPATH=/etc/firefly/organizations/peerOrganizations/org1.example.com/users/Admin@org1.example.com/msp",
+		"-v", fmt.Sprintf("%s:/package.tar.gz", packageFilename),
+		"-v", fmt.Sprintf("%s:/etc/firefly", volumeName),
+		"hyperledger/fabric-tools:2.3",
```

Comment:
We have this version string in quite a lot of places in the code, wonder if it should be pulled out?

Reply:
Assume this is deferred. Should we track it somewhere?